src/views/PageOrgchart.vue

```diff
@@ -94,9 +93,8 @@ export default {
       this.loading = false;
     },
     honourUriFragments() {
-      if (window.location.hash) {
-        this.$store.commit('updatePreviewProfileId', {
-          newId: window.location.hash.substr(1),
+      if (this.$route.params.userId) {
```

Comment:
I think we can get rid of `honourUriFragments`; as I understand it we navigate using a route, and the routes are already being honoured.

There's no need to send `document.body` as the trigger; the purpose for it was that focus would get returned to the trigger, but focus gets returned to the body by default.